modules/apps/depot/depot-service/src/main/java/com/liferay/depot/service/impl/DepotEntryLocalServiceImpl.java

```diff
@@ -110,7 +112,8 @@ public DepotEntry updateDepotEntry(
 			ServiceContext serviceContext)
 		throws PortalException {
 
-		DepotEntry depotEntry = getDepotEntry(depotEntryId);
+		DepotEntry depotEntry = depotEntryPersistence.update(
```

Comment:
We only update the depot entry here so that the modified date is correct. All other data is stored in the Group model.